renku/ui/service/views/apispec.py

```diff
@@ -78,7 +78,7 @@ def path_helper(self, path, operations, *, view, app=None, **kwargs):
     openapi_version=OPENAPI_VERSION,
     version=API_VERSION,
     plugins=[MultiURLFlaskPlugin(), MarshmallowPlugin()],
-    servers=[{"url": SERVICE_API_BASE_PATH}],
+    servers=[{"url": SERVICE_API_BASE_PATH}, {"url": "/ui-server"+SERVICE_API_BASE_PATH}],
```

Comment:
We can write this a f-string and avoid using string addition: `f"/ui-server{SERVICE_API_BASE_PATH}"`

Reply:
Sure I will make the change. Just FYI this is still WIP and a draft.